### ISSUE TITLE
Missing heading ARIA

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-navigationTitle#5]
Currently, the title is readable but there is no heading role or level associated so this is just read as standard text.

If the title is configured to use the [course title ](https://github.com/cgkineo/adapt-navigationTitle/blob/da0695990c6f3bf0f1abb0985832a025ae7092bc/example.json#L10), seeing as this is already announced on course load as site `<title>`, I think we could just set this to `aria-hidden` to save repetition?

In the instance where the title differs from the [course title](https://github.com/cgkineo/adapt-navigationTitle/blob/da0695990c6f3bf0f1abb0985832a025ae7092bc/example.json#L4), I think this would need to be marked up with the appropriate aria if the title isn't included elsewhere (not included in page or menu title etc). Making this `aria-level=1` would have a knock on effect on other titles however as page and menu would become `aria-level=2` but this can be manually configured in the [_ariaLevels config](https://github.com/adaptlearning/adapt_framework/pull/3319/files).
